prow/cmd/phaino/local.go

```diff
@@ -85,6 +85,9 @@ func readRepo(ctx context.Context, path string) (string, error) {
 		return "", fmt.Errorf("workingDir: %v", err)
 	}
 	def, err := findRepo(wd, path)
+	if err != nil { // If k8s/test-infra is not under GOPATH, find under GOPATH.
+		def, err = findRepo(filepath.Join(os.Getenv("GOPATH"), "src"), path)
```

Comment:
GOPATH env doesn't have to be set to have a GOPATH (it has a default, try `go env GOPATH` in an environment without the env, it will be ~`~/go`).

you can locate a package how go does with: https://golang.org/pkg/go/build/
```golang
// pkg.Dir will be the path if found
pkg, err := build.Default.Import(path, build.Default.GOPATH, build.FindOnly|build.IgnoreVendor)
```

Reply:
Great! Done.